exodus/trackers/tests.py

```diff
@@ -95,11 +95,12 @@ def test_should_return_stats_with_multiple_trackers_found(self):
         self.assertContains(response, tracker1.name, 1)
         self.assertContains(response, tracker2.name, 1)
         self.assertEqual(response.context['trackers'][0].name, tracker2.name)
-        self.assertEqual(response.context['trackers'][0].count, 2)
-        self.assertEqual(response.context['trackers'][0].score, 66)
+        # Only recent for an application is considered
+        self.assertEqual(response.context['trackers'][0].count, 1)
+        self.assertEqual(response.context['trackers'][0].score, 100)
         self.assertEqual(response.context['trackers'][1].name, tracker1.name)
         self.assertEqual(response.context['trackers'][1].count, 1)
-        self.assertEqual(response.context['trackers'][1].score, 33)
+        self.assertEqual(response.context['trackers'][1].score, 100)
```

Comment:
I think we need to diversify the tests a bit more, with different cases:
1) only reports with different handles
2) reports with the same handle to make sure it only considers the last one

Do you feel like writing those or do you want me to take a look?

Reply:
I'll try writing. If I have any doubts I'll let you know.

Reply:
@simpnu Can you verify the updated test cases? And if this looks fine can you suggest me some other task I can take up?